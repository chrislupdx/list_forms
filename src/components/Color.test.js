import React from 'react';
import { shallow } from 'enzyme';
import Color from './Color';

describe('Color component', () => {
  it('renders a color', () => {
    const rgb = {
      red: 111,
      green: 3,
      blue: 0
    };

    const wrapper = shallow(<Color name="Red" rgb={rgb} />);
    expect(wrapper).toMatchSnapshot();
  });
})
;
