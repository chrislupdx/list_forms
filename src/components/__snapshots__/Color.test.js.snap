// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Color component renders a color 1`] = `
<d1>
  <dt>
    Name
  </dt>
  <dd>
    Red
     
    <div
      style={
        Object {
          "backgroundColor": "#6F0300",
          "display": "inline-block",
          "height": "0.8em",
          "width": "0.8em",
        }
      }
    />
  </dd>
  <dt>
    Hex
  </dt>
  <dd>
    #6F0300
  </dd>
  <dt>
    RGB
  </dt>
  <dd>
    <p>
      Red - 
      111
       
    </p>
    <p>
      Blue - 
      0
       
    </p>
    <p>
      Green - 
      3
       
    </p>
  </dd>
</d1>
`;
